modules/govuk/manifests/node/s_logs_elasticsearch.pp

```diff
@@ -35,6 +35,24 @@
     ],
   }
 
+  case $govuk_elasticsearch::version {
+    /^0.9/: { $cloud_aws_version = '1.16.0' }
+    /^1.0/: { $cloud_aws_version = '2.0.0' }
+    /^1.1/: { $cloud_aws_version = '2.1.1' }
+    /^1.2/: { $cloud_aws_version = '2.2.0' }
+    /^1.3/: { $cloud_aws_version = '2.3.0' }
+    /^1.4/: { $cloud_aws_version = '2.4.2' }
+    /^1.5/: { $cloud_aws_version = '2.5.1' }
+    /^1.6/: { $cloud_aws_version = '2.6.1' }
+    /^1.7/: { $cloud_aws_version = '2.7.1' }
+    default: { $cloud_aws_version = '2.7.1' }
+  }
```

Comment:
There's no need to set this for values below 1.4 because we'll never run those versions of Elasticsearch on our logs cluster. We could also set the default case to call `fail()` so that we don't install a potentially breaking version of the plugin if we upgrade our cluster.